tbc_error_page/error.json

```diff
@@ -0,0 +1,194 @@
+(lp1
```

Comment:
Not really, but its here working as a dummy. Will be removed if unecessary. Also, these are not really json files. Pickled files. My bad. Will change the names to suit the content